clarity_ext/domain/result_file.py

```diff
@@ -1,35 +1,35 @@
-from clarity_ext.unit_conversion import UnitConversion
-from clarity_ext.domain.artifact import Artifact
+from clarity_ext.domain.aliquot import Aliquot, Sample
 
 
-class ResultFile(Artifact):
+class ResultFile(Aliquot):
```

Comment:
Why does it now inherit form Aliquot? It seems unnecessarily specific to me, since it can be any file - perhaps not in the step you're working with, but generally. See definition here:

"ResultFile: The process outputs a file, such as a result file or a file produced by an instrument."
https://genologics.zendesk.com/hc/en-us/articles/213973183-Creating-Configuring-Processes

The reason for why it is inheriting from `Artifact` was that it needs to know if it's an input artifact and have UDFs associated to it.